librz/core/cmd/cmd_print.c

```diff
@@ -1347,8 +1348,7 @@ static void annotated_hexdump(RzCore *core, int len) {
 	bytes = calloc(nb_cons_cols * 40, sizeof(char));
 	if (!bytes)
 		goto err_bytes;
-#if 1
-	int addrpadlen = strlen(sdb_fmt("%08" PFMT64x, addr)) - 8;
+	int addrpadlen = strlen(rz_strf(tmpbuf, "%08" PFMT64x, addr)) - 8;
```

Comment:
i would convert this into `snprintf` because this function actually returns the length of the written text.

Reply:
Not really; the result is different. I am keeping the old variant.